src/test/test_bitcoin.cpp

```diff
@@ -68,7 +68,7 @@ TestingSetup::TestingSetup(const std::string& chainName) : BasicTestingSetup(cha
 
         RegisterAllCoreRPCCommands(tableRPC);
         ClearDatadirCache();
-        pathTemp = fs::temp_directory_path() / strprintf("test_bitcoin_%lu_%i", (unsigned long)GetTime(), (int)(InsecureRandRange(100000)));
+        fs::path pathTemp = util_tests::unit_test_directory("test_bitcoin");
```

Comment:
This shadows `pathTemp`, you might as well rename it `m_path_temp` according to the developer guidelines